blog/macos-rig/index.qmd

```diff
@@ -0,0 +1,309 @@
+* There's even a plugin that reminds you what your aliases are! -- for maximum discoverability: [https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/aliases](https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/aliases)
+
+You can also hand-code your themes into your `.zshrc` dotfile instead of using the OMZ themes. The aliases not provided by OMZ can be added to the custom.zsh dotfile, such as the ones in the dotfiles linked below.
+
+Install OMZ with:
+
+```bash
+sh -c "$(curl -fsSL https://raw.github.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"
+```
+
+6. Configure [Oh My Zsh](https://github.com/gvelasq/dotfiles/tree/main/zsh).
+
+To configure oh-my-zsh, you can:
+
+1. **Edit your settings: **Navigate to your home directory `~/` and edit the `.zshrc` file.
+1. **Use preconfigured settings: **Navigate to your home directory `~/`. Replace the default `.zshrc` file with [this file](https://github.com/gvelasq/dotfiles/blob/main/zsh/.zshrc). Then, navigate to the hidden folder `~/.oh-my-zsh/custom` and drop [this file](https://github.com/gvelasq/dotfiles/blob/main/zsh/custom.zsh) in the folder.
+
+:::{.callout-tip}
+On a Mac, you can press Command + Shift + . to see your hidden folders.
+:::
+
+A `custom.zsh` file might be helpful because you can add many more commands for OMZ plugins than what are built in. For instance,
+
+```bash
+alias bog="brew outdated --greedy"
+```
+
+This allows you to use `bog` to run `brew outdated --greedy`, instead of having to type the whole command out. That particular command is helpful if you want to find ALL the programs that could potentially be upgraded.
+
+![](image1.jpg){fig-align="center" fig-alt="Keep Calm and bubo bog bupc bc" fig-width="400"}
+
+7. Go back to the Terminal and install [rig](https://github.com/r-lib/rig#linux).
+
+[rig](https://github.com/r-lib/rig#linux), the 'R Installation Manager,' is an amazing tool. It not only lets you download R, but install, configure, switch between, and remove _multiple_ R versions. It also allows you to manage _user-level package libraries_ (meaning you can delete a version of R without losing all your packages!). Mara Averick wrote a great summary on [Day 10 of Tidy Advent 2022](https://mara.quarto.pub/tidy-advent-2022/posts/10-rig.html).
+
+Install rig by running these commands in the Terminal:
+
+```bash
+brew tap r-lib/rig
+brew install --cask rig
+```
+
+8. Install R.
+
+Install the latest version of R using rig:
+
+```bash
+rig add release
+```
+
+9. Install [RStudio Desktop](https://posit.co/download/rstudio-desktop).
+
+You want the latest and greatest RStudio, no? Use Homebrew to download and install the latest version. Once a new version is released, the community is very quick at updating the cask entry for RStudio in Homebrew.
+
+```bash
+brew install --cask rstudio
+```
+
+10. <a name="abcd">Configure [`.Rprofile`](https://github.com/gvelasq/dotfiles/blob/main/R/.Rprofile).</a>
```

Comment:
Why is there HTML around this line? `<a name="abcd"></a>`

Reply:
To create an anchor link. I don't know how to do it in Markdown.